platform/8.0/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Jakarta Platform Spec 8.0"
+date: 2019-07-18T16:10:38-04:00 (use "date +%FT%T%z")
```

Comment:
Don't forget to remove the "use" comment at the end for the final version.

Reply:
got it, thanks